README.md

```diff
@@ -33,23 +37,61 @@ This outputs
 AST: [Query(Query { ctes: [], body: Select(Select { distinct: false, projection: [UnnamedExpr(Identifier("a")), UnnamedExpr(Identifier("b")), UnnamedExpr(Value(Long(123))), UnnamedExpr(Function(Function { name: ObjectName(["myfunc"]), args: [Identifier("b")], over: None, distinct: false }))], from: [TableWithJoins { relation: Table { name: ObjectName(["table_1"]), alias: None, args: [], with_hints: [] }, joins: [] }], selection: Some(BinaryOp { left: BinaryOp { left: Identifier("a"), op: Gt, right: Identifier("b") }, op: And, right: BinaryOp { left: Identifier("b"), op: Lt, right: Value(Long(100)) } }), group_by: [], having: None }), order_by: [OrderByExpr { expr: Identifier("a"), asc: Some(false) }, OrderByExpr { expr: Identifier("b"), asc: None }], limit: None, offset: None, fetch: None })]
 ```
 
+## SQL compliance
+
+SQL was first standardized in 1987, and revisions of the standard have been
+published regularly since. Most revisions have added significant new features to
+the language, and as a result no database claims to support the full breadth of
+features. This parser currently supports most of the SQL-92 syntax, plus some
+syntax from newer versions that have been explicitly requested, plus some MSSQL-
+and PostgreSQL-specific syntax. Whenever possible, the [online SQL:2011
+grammar][sql-2011-grammar] is used to guide what syntax to accept. (We will
+happily accept changes that conform to the SQL:2016 syntax as well, but that
+edition's grammar is not yet available online.)
+
+Unfortunately, stating anything more specific about compliance is difficult.
+There is no publicly available test suite that can assess compliance
+automatically, and doing so manually would strain the project's limited
+resources. Still, we are interested in eventually supporting the full SQL
+dialect, and pull requests that improve compliance will almost certainly be
+accepted.
+
+If you are assessing whether this project will be suitable for your needs,
+you'll likely need to experimentally verify whether it supports the subset of
+SQL that you need. Please file issues about any unsupported queries that you
+discover. Doing so helps us prioritize support for the portions of the standard
+that are actually used.
+
+### Supporting custom SQL dialects
+
+This is a work in progress, but we have some notes on [writing a custom SQL
+parser](docs/custom_sql_parser.md).
+
 ## Design
 
-This parser is implemented using the [Pratt Parser](https://tdop.github.io/) design, which is a top-down operator-precedence parser.
+This parser is implemented using the [Pratt Parser] design, which is a top-down
```

Comment:
I didn't read the paper carefully, but I believe the Pratt parser is what we use to parse expressions (I found https://eli.thegreenplace.net/2010/01/02/top-down-operator-precedence-parsing more useful to understand the technique), otherwise what we have is hand-coded recursive descent parser. Right?

Reply:
Yes, all true. Updated accordingly.